packages/app/src/containers/ComponentsWindow.tsx

```diff
@@ -0,0 +1,91 @@
+import Sketch from '@sketch-hq/sketch-file-format-ts';
+import { memo, useCallback } from 'react';
+import ColorSwatch from '../components/swatches/ColorSwatch';
+import ColorSwatchGrid from './ColorSwatchGrid';
+import {
+  useApplicationState,
+} from '../contexts/ApplicationStateContext';
+import {
+  getSharedSwatches
+} from 'noya-state/src/selectors';
+import styled from 'styled-components';
+
+/**
+ * TODO: Multiselect and change color 
+ * TODO: Clean code
+ */
+
+const SwatchContainer = styled.div`
+  height: 175px,
+  &:hover {
```

Comment:
This would be `"&:hover": { ... }`, although I think in this case we don't need the `:hover`, since the cursor property only matters when the element is hovered anyway